### ISSUE TITLE
How about join the multi-picture selection?

### ISSUE THREAD

**Issue by LuckyZC** [memspace/zefyr#22]
In many cases, we need to use multi-picture selection

**Comment by pulyaevskiy** [comment]
Good question. I thought about this as well.

Curious what would be expected result in your case? With existing image support it would mean that images would be inserted one after another on separate lines. Is it how you expect multi-picture to work?

**Comment by LuckyZC** [comment]
Yes, many articles require continuous multi-picture display, and each picture can be satisfied with a single line.

**Comment by pinksmallpig** [comment]
1.edit [pickImage()](https://github.com/memspace/zefyr/blob/7f102b9943ae37c6520d081a2b084d704bfa90b3/packages/zefyr/lib/src/widgets/image.dart#L39) return `Future<List<String>>`
2.edit [_pickFromGallery()](https://github.com/memspace/zefyr/blob/7f102b9943ae37c6520d081a2b084d704bfa90b3/packages/zefyr/lib/src/widgets/buttons.dart#L288)  with below code
`final editor = ZefyrToolbar.of(context).editor;`
 `    final images = await editor.imageDelegate.pickImage(editor.imageDelegate.gallerySource);`
 `   for (var image in images) {`
`        editor.formatSelection(NotusAttribute.embed.image(image));`
`    }`
